src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -934,7 +934,11 @@ public boolean verifyVisibilityMessageIsDisplayed(int questionNumber) {
     public boolean verifyVisibilityOptionsIsDisplayed(int questionNumber) {
         return getVisibilityOptions(questionNumber).isDisplayed();
     }
-    
+
+    public WebElement getVisibilityOptionTableRow(int questionNumber, int optionRowNumber) {
+        return getVisibilityOptions(questionNumber).findElement(By.xpath("table/tbody/tr[" + optionRowNumber + "]"));
```

Comment:
@Sumei1009, Are you still using this xpath selector? You mentioned using a css selector. Both are fine, but do push your updated code.